clarify in the docs that keys in the product field can be anything

I got a user who asked me if their `pipeline.yaml` looked well and I noticed they had something like this:

```
tasks:
   # ... a bunch of tasks
   - source: fit.py
      product:
        nb: path/to/report.html
        data: path/to/data.csv
        data1: path/to/data.csv
        data2: path/to/data.csv
```

The user was confused and thought that the naming should be name, name1, name2. So we should add a little note in the basic concepts guide to explicitly say that keys can be anything

https://docs.ploomber.io/en/latest/get-started/basic-concepts.html

### Error building doc
I followed the instructions in [CONTRIBUTING.doc](https://github.com/mhered/ploomber/blob/master/doc/CONTRIBUTING.md) but cannot build the docs:
```
...
$ invoke setup --doc
...
Obtaining file:///home/mhered/hacktoberfest22/ploomber/tests/assets/test_pkg
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Installing collected packages: test-pkg
  Running setup.py develop for test-pkg
Successfully installed test-pkg-0.0.0

ArgumentError: activate does not accept more than one argument:
['ploomberconda', 'env', 'update', '--file', 'environment.yml', '--name', 'ploomber']

$ conda activate ploomber
(ploomber) $ invoke docs
Running Sphinx v5.2.3

Extension error:
Could not import extension nbsphinx (exception: No module named 'nbsphinx')
make: *** [Makefile:23: html] Error 2
(ploomber) $ 
```

--- a/doc/api/spec.rst
+++ b/doc/api/spec.rst
@@ -597,7 +597,9 @@ If the task generates multiple products, pass a dictionary:
             data: output/data.csv
 
 
-The mechanism to make ``product`` available when exeuting your task
+.. note:: The name of keys in the product dictionary can be chosen freely so as to be descriptive of the outputs (e.g. ``data``, ``data_clean``, ``model``, etc.)
+
+The mechanism to make ``product`` available when executing your task
 depends on the type of task.
 
 SQL tasks receive a ``{{product}}`` placeholder:

--- a/doc/get-started/basic-concepts.rst
+++ b/doc/get-started/basic-concepts.rst
@@ -126,7 +126,7 @@ copy of your scripts, converts them to .ipynb at runtime and executes them.** Th
 crucial concept: scripts are part of your project's source code, but executed
 notebooks are pipeline products.
 
-.. note:: Even though we recommend the use of ``.py`` files, you you can still use regular ``.ipynb`` files as sources if you prefer so.
+.. note:: Even though we recommend the use of ``.py`` files, you can still use regular ``.ipynb`` files as sources if you prefer so.
 
 To know more about integration with Jupyter notebooks, see the :doc:`../user-guide/jupyter` guide.
 
@@ -171,7 +171,7 @@ The cell injection process
     For tips on troubleshooting pipeline loading, see :ref:`Troubleshooting pipeline loading`.
     
 
-Let's review the contents a sample ``clean.py`` file:
+Let's review the contents of a sample ``clean.py`` file:
 
 .. code-block:: python
    :class: text-editor
@@ -217,7 +217,7 @@ The cell injection process also happens when opening the notebook/script in Jupy
 .. note::
    
    When using ``jupyter notebook``, scripts open automatically as
-   notebooks. If using ``jupyter lab``, you have to click right and select the
+   notebooks. If using ``jupyter lab``, you have to right-click and select the
    notebook option.
 
 Since scripts/notebooks always create an executed notebook, you must specify
@@ -245,6 +245,7 @@ dictionary under ``product``:
           # ...and any other keys for other files
           data: output/data.csv
 
+.. note:: The name of keys in the product dictionary can be chosen freely so as to be descriptive of the outputs (e.g. ``data``, ``data_clean``, ``model``, etc.)
 
 This covers scripts and notebooks as tasks, if you want to learn how to use
 functions as tasks, keep scrolling, otherwise, :ref:`skip to the end. <where-to-go-from-here>`

